src/applications/disability-benefits/781/pages/ptsdSecondaryChoice.js

```diff
@@ -0,0 +1,70 @@
+import React from 'react';
+import {
+  ptsdNameTitle,
+} from '../helpers';
+import AdditionalInfo from '@department-of-veterans-affairs/formation/AdditionalInfo';
+
+const uploadPtsdSecondaryDescription = () => {
+  return (
+    <div>
+      <p>
+        The following questions will help us understand more about your
+        [PTSDclassification]-related PTSD. None of the questions we'll ask you are
```

Comment:
Claire from AdHoc told me to use curly quote: ‘. Alt + ]

Reply:
See above.

Reply:
More 'special' quotes?